robottelo/ui/locators/base.py

```diff
@@ -478,34 +473,34 @@
     ),
     "resource.compute_profile.storage_add_node": (
         By.XPATH,
-        ("//div/fieldset[@id='storage_volumes']"
-         "/a[@data-association='volumes']")
+        ("//*[@id='storage_volumes']"
```

Comment:
is there a reason to use `*` here?

Reply:
No, Updated the locator with element to follow instead of wildcard